wagtail/core/wagtail_hooks.py

```diff
@@ -284,9 +284,10 @@ class DeleteViewRestrictionActionFormatter(LogFormatter):
 
         def format_message(self, log_entry):
             try:
-                return _("Removed the '%(restriction)s' view restriction") % {
+                return _("Removed the '%(restriction)s' view restriction. The page is public now.") % {
```

Comment:
```suggestion
                return _("Removed the '%(restriction)s' view restriction. The page is public.") % {
```

let's avoid confusion